<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DATVModGUI</class>
 <widget class="RollupContents" name="DATVModGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>235</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>DATV Modulator</string>
  </property>
  <widget class="QWidget" name="settingsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>541</width>
     <height>221</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="deltaFreqPowLayout">
      <item>
       <layout class="QHBoxLayout" name="deltaFrequencyLayout">
        <property name="topMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="deltaFrequencyLabel">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Df</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ValueDialZ" name="deltaFrequency" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Liberation Mono</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="toolTip">
           <string>Demod shift frequency from center in Hz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="deltaUnits">
          <property name="text">
           <string>Hz </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="channelSampleRateText">
          <property name="minimumSize">
           <size>
            <width>66</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Sample rate</string>
          </property>
          <property name="text">
           <string>00000.00k</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dataRateText">
          <property name="toolTip">
           <string>Data rate of DVB stream</string>
          </property>
          <property name="text">
           <string>0kb/s</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_12">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="channelPower">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Channel power</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>-100.0 dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="channelMute">
        <property name="toolTip">
         <string>Mute/Unmute channel</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/txon.png</normaloff>
          <normalon>:/txoff.png</normalon>:/txon.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="rfBandwidthLayout">
      <item>
       <widget class="QComboBox" name="dvbStandard">
        <property name="toolTip">
         <string>DVB standard</string>
        </property>
        <item>
         <property name="text">
          <string>DVB-S</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DVB-S2</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_15">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Symbols/s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="symbolRate">
        <property name="toolTip">
         <string>Symbol rate</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="value">
         <number>250000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rfBWLabel">
        <property name="text">
         <string>BW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="rfBW">
        <property name="toolTip">
         <string>Modulator (RF) bandwidth when interpolation takes place</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rfBWText">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0000k</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="volumeLayout_2">
      <item>
       <widget class="QLabel" name="inputSelectLabel">
        <property name="text">
         <string>Source</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="inputSelect">
        <property name="toolTip">
         <string>Source of MPEG transport stream to transmit</string>
        </property>
        <item>
         <property name="text">
          <string>File</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UDP</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fecLabel">
        <property name="text">
         <string>FEC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fec">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Forward error correction code rate</string>
        </property>
        <item>
         <property name="text">
          <string>1/2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2/3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3/4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5/6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7/8</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="modulationLabel">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Modulation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="modulation">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Modulation type</string>
        </property>
        <item>
         <property name="text">
          <string>BPSK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QPSK</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rollOffLabel">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Roll off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="rollOff">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Root raised cosine filter roll-off</string>
        </property>
        <item>
         <property name="text">
          <string>0.35</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_nav_2">
      <item>
       <widget class="QLabel" name="udpPortLabel">
        <property name="text">
         <string>UDP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="udpAddress">
        <property name="toolTip">
         <string>IP address to bind UDP socket to</string>
        </property>
        <property name="inputMask">
         <string>000.000.000.000</string>
        </property>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="udpPort">
        <property name="toolTip">
         <string>UDP port number to receive MPEG transport stream on</string>
        </property>
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>5004</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="udpBitrate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>UDP bitrate</string>
        </property>
        <property name="text">
         <string>0kb/s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="udpBufferUtilizationLine">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="udpBufferUtilization">
        <property name="toolTip">
         <string>Indicates how full the UDP receive buffer is. If this reaches 100%, packets will likely be dropped.</string>
        </property>
        <property name="text">
         <string>0%</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="tsFileLayout">
      <item>
       <widget class="QPushButton" name="tsFileDialog">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Open MPEG transport stream file</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/film_reel.png</normaloff>:/film_reel.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tsFileText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tsFileBitrate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>File bitrate</string>
        </property>
        <property name="text">
         <string>0kb/s</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="playControllLayout">
      <item>
       <widget class="ButtonSwitch" name="playLoop">
        <property name="toolTip">
         <string>Play file in a loop</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/playloop.png</normaloff>:/playloop.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="playFile">
        <property name="toolTip">
         <string>File play/pause</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/play.png</normaloff>
          <normalon>:/pause.png</normalon>
          <disabledoff>:/play.png</disabledoff>
          <disabledon>:/play.png</disabledon>:/play.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="linePlay1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="relTimeText">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Time from start</string>
        </property>
        <property name="text">
         <string>00:00:00.000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="linePlay2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="recordLengthText">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Total time</string>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_nav">
      <item>
       <widget class="QSlider" name="navTimeSlider">
        <property name="toolTip">
         <string>File time navigator</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupContents</class>
   <extends>QWidget</extends>
   <header>gui/rollupcontents.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ValueDialZ</class>
   <extends>QWidget</extends>
   <header>gui/valuedialz.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
